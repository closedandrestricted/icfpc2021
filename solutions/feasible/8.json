{"vertices":[[43,94],[51,78],[56,77],[37,65],[32,55],[18,48],[85,61],[34,39],[29,41],[42,41],[42,25],[55,27],[90,19]]}