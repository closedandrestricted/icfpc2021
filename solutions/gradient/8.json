{"vertices":[[45,92],[53,76],[58,75],[36,64],[31,53],[18,47],[84,61],[34,38],[29,40],[42,38],[40,22],[53,24],[90,19]]}