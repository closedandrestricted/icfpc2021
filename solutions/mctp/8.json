{"vertices":[[79,26],[62,28],[58,30],[68,47],[75,57],[76,71],[22,45],[68,69],[64,67],[58,55],[51,67],[46,79],[37,8]]}