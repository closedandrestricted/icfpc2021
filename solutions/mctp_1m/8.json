{"vertices":[[43,17],[50,33],[54,31],[40,33],[30,40],[16,49],[80,39],[30,54],[26,56],[39,51],[39,66],[47,74],[88,67]]}