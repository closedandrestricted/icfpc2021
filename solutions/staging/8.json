{"vertices":[[36,17],[38,35],[34,37],[49,64],[62,63],[76,60],[10,52],[71,50],[76,48],[71,37],[57,44],[54,32],[43,93]]}