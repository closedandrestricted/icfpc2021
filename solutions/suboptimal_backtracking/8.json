{"vertices":[[7,53],[21,45],[25,43],[33,50],[35,63],[44,75],[37,7],[48,66],[53,68],[67,70],[58,56],[62,44],[90,19]]}