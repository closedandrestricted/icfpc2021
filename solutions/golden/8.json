{"vertices":[[7,53],[21,45],[25,43],[34,49],[34,62],[44,75],[36,6],[48,66],[53,68],[67,70],[58,56],[62,44],[90,19]]}